Create docs (#124)
tomchor/Oceanostics.jl: docs/src/index.md

```@eval
using Markdown
Markdown.parse_file(joinpath(@__DIR__, "..", "..", "README.md"))
```

!!! note "This documentation is still under construction"
    We are still working on these docs. If you see any errors or if you have any helpful suggestions please 
    open [an issue](https://github.com/tomchor/Oceanostics.jl/issues/new) or
    [a pull request](https://github.com/tomchor/Oceanostics.jl/pulls) on github.

## Quick example

> The example below is a simple illustration of how to use a few of Oceanostics features:

Reviewer: ```suggestion
This example illustrates a few of Oceanostics features:
```
